src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -384,6 +386,34 @@ TEST_F(LiveFunctionsDataViewTest, ContextMenuActionsAreInvoked) {
     CheckExportToCsvIsInvoked(context_menu, app_, view_, expected_contents);
   }
 
+  // Export events to CSV
+  {
+    const uint32_t kThreadId = 111;
+    const std::string kThreadName = "Test Thread";
+    capture_data_->AddOrAssignThreadName(kThreadId, kThreadName);
+    EXPECT_CALL(app_, GetCaptureData).WillRepeatedly(testing::ReturnRef(*capture_data_));
+
+    constexpr uint64_t kStart = 12345;
+    constexpr uint64_t kEnd = 67890;
+    TimerInfo timer;
```

Comment:
Maybe timer here is a good idea. Maybe one thread with 1 timer and one thread with 2 timers

Reply:
Done! Thanks!